albo.py

```diff
@@ -12,23 +12,30 @@ def escape_char (text: str, char_to_escape: List[str] = ['_', '*', '[', '`']) ->
     text = text.replace(char, "\\" + char)
   return text
 
+def send_https_request (URL: str) -> None:
+  r = requests.get(url = URL)
+  responseJSON = r.json()
+  if responseJSON["ok"] == False:
+    raise ValueError (json.dumps(responseJSON, indent=2), URL.split("&")[-1])
+
+
 def send_telegram_message(text: str) -> None:
```

Comment:
```suggestion
def send_telegram_message(text: str, chat_id = CHATID) -> None:
```

Reply:
Might as well update all the other methods to have `chat_id` as an argument, for consistency